elasticsearch/examples/kubesphere/README.md

```diff
@@ -226,3 +159,84 @@ $ curl http://<your node ip>:30057/_cluster/health?pretty
   "active_shards_percent_as_number" : 100.0
 }
 ```
+
+### The reference configuration
+
+#### Master node
+
+```
+clusterName: "elasticsearch"
+nodeGroup: "master"
+
+roles:
+  master: "true"
+  ingest: "false"
+  data: "false"
+  ml: "false"
+  remote_cluster_client: "false"
+
+# tolerations for master node
+tolerations:
+- effect: NoSchedule
+  key: node-role.kubernetes.io/master
+  value: ''
+
+replicas: 3
+minimumMasterNodes: 2
+
+esJavaOpts: "-Xmx512m -Xms512m"
+
+resources:
+  requests:
+    cpu: "512m"
+    memory: "512Mi"
+  limits:
+    cpu: "1"
+    memory: "1Gi"
+
+```
+
+#### Data node
+
+```
+clusterName: "elasticsearch"
+nodeGroup: "data"
+
+roles:
+  master: "false"
+  ingest: "true"
+  data: "true"
+  ml: "false"
+  remote_cluster_client: "false"
+
+
+# tolerations for data node
+tolerations:
+- effect: NoSchedule
+  key: node-role.kubernetes.io/master
+  value: ''
+
+# replicas 
+replicas: 3
+minimumMasterNodes: 2
+
+# Starting from version 7.11, Elasticsearch automatically sizes JVM heap based on a node's roles and total memory.
+# This configuration should not be modified.
+esJavaOpts: null
+
+# Adjust the limit of the Data node according to the data size, and the memory can be up to 32GI
```

Comment:
Adjust the limit of the Data node according to the data size, and the memory can be up to 32GI
=>
Adjust the limit of the Data node according to the data size, the memory can be set up to 32Gi.